integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestLoadDataWithDiffTimestampFormat.scala

```diff
@@ -354,6 +355,7 @@ class TestLoadDataWithDiffTimestampFormat extends QueryTest with BeforeAndAfterA
 
   test("test load, update data with setlenient session level property for daylight " +
        "saving time from different timezone") {
+    TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
```

Comment:
If the above comment works, you can remove this

Reply:
done